canonical/ubuntu.com
# fix: accessibility for contact modals

## Done

- This fix accessibility within all contact modals.

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes

- Go to this page http://127.0.0.1:8001/kubernetes and click on CTA button, when the modal is closed make sure the CTA is focused.
- Please delete all cookies to test cookie modal and contact modal, when we open this url [http://127.0.0.1:8001/kubernetes#get-in-touch](http://127.0.0.1:8001/kubernetes#get-in-touch)

## Issue / Card

Fixes #12002 

## Screenshots

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
